ncm-network/src/main/perl/network.pm

```diff
@@ -570,6 +623,56 @@ sub Configure
             };
             $exifiles{$file_name} = $self->file_dump($file_name, $text, FAILED_SUFFIX);
             $self->debug(3, "exifiles $file_name has value ".$exifiles{$file_name});
+        } elsif (exists($net{$iface}{policy_route})) {
+            $file_name = "$dir_pref/route-$iface";
+            $exifiles{$file_name} = 1;
+            $text="";
+
+            # We can't mix different syntaxes in the same file, so convert
+            # simple routes to the policy routing syntax
+            if (exists($net{$iface}{route})) {
+                foreach my $rt (sort keys %{$net{$iface}{route}}) {
+                    my $route = {"dev" => $iface};
```

Comment:
this is not correct, should be devicename like https://github.com/quattor/configuration-modules-core/pull/1087/files#diff-90bc6b2e2a105ae5c62daf55ef95b2a7R1573

but new code makes this much easier; and also support arbitrary commands via `cmd` attribute